src/components/ui/icons.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  ChevronRightIcon,
+  EllipsisHorizontalIcon,
+  CheckIcon,
+  XCircleIcon,
+  XMarkIcon,
+  MagnifyingGlassIcon,
+  Bars3BottomLeftIcon,
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+  MinusIcon,
+  Bars3Icon,
+  ChevronLeftIcon,
+  DocumentIcon,
+  PencilIcon,
+  EyeIcon,
+  QuestionMarkCircleIcon,
+  CogIcon,
+  UserIcon,
+  ArrowRightOnRectangleIcon,
+  CalendarIcon,
+  StarIcon,
+  ArrowDownTrayIcon,
+  HomeIcon,
+  UsersIcon,
+  FolderOpenIcon,
+  ChevronUpDownIcon,
+} from '@heroicons/react/24/solid';
+
+export {
+  ChevronRightIcon as ChevronRight,
+  EllipsisHorizontalIcon as MoreHorizontal,
+  CheckIcon,
+  XCircleIcon as CircleIcon,
+  XMarkIcon as XIcon,
+  MagnifyingGlassIcon as SearchIcon,
+  Bars3BottomLeftIcon as PanelLeftIcon,
+  ArrowLeftIcon as ArrowLeft,
+  ArrowRightIcon as ArrowRight,
+  ChevronDownIcon,
+  ChevronUpIcon,
+  MinusIcon,
+  Bars3Icon as GripVerticalIcon,
+  ChevronLeftIcon,
+  EllipsisHorizontalIcon as MoreHorizontalIcon,
+  DocumentIcon as FileIcon,
+  PencilIcon,
+  EyeIcon,
+  QuestionMarkCircleIcon as HelpCircleIcon,
+  CogIcon as SettingsIcon,
+  UserIcon,
+  ArrowRightOnRectangleIcon as LogOutIcon,
+  CalendarIcon,
+  StarIcon,
+  ArrowDownTrayIcon as DownloadIcon,
+  HomeIcon as Home,
+  UsersIcon as Users,
+  FolderOpenIcon,
+  ChevronDownIcon as ChevronDown,
+  ChevronUpDownIcon as ChevronsUpDown,
+  CheckIcon as Check,
+}; 
```

Comment:
Naming here looks inconsistent.. prolly a good idea to either always include `*Icon` or remove it.